makefile

```diff
@@ -119,3 +119,23 @@ flow-typed: yarn.lock
 	@cd packages/guui && ../../node_modules/.bin/flow-typed install -p ../../
 	@cd packages/pasteup && ../../node_modules/.bin/flow-typed install -p ../../
 	@flow-typed install
+
+# packages #########################################
+
+clean-pasteup:
+	@rm -rf packages/pasteup/dist packages/pasteup/src
+
+pre-publish-pasteup:
+	$(call log, "building pasteup")
+	@mkdir packages/pasteup/src
```

Comment:
Why is it necessary to create a `src` directory, copy files to that directory and use them in the build and afterwards remove that `src` directory?

Reply:
We could rename src to temp of it makes this intention clearer

Reply:
yep I definitely think renaming it `temp` is more appropriate. `src` has other connotations in the realm of packages/libraries.